pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -211,12 +206,14 @@ buildPythonPackage {
     scipy
   ];
 
-  # Note that cudatoolkit is snecessary since jaxlib looks for "ptxas" in $PATH.
-  # See https://github.com/NixOS/nixpkgs/pull/164176#discussion_r828801621 for
-  # more info.
+  # jaxlib looks for ptxas at runtime, eg when running `jax.random.PRNGKey(0)`.
+  # Linking into $out is the least bad solution. See
+  # * https://github.com/NixOS/nixpkgs/pull/164176#discussion_r828801621
+  # * https://github.com/NixOS/nixpkgs/pull/288829#discussion_r1493852211
+  # for more info.
   postInstall = lib.optional cudaSupport ''
-    mkdir -p $out/bin
-    ln -s ${cudatoolkit}/bin/ptxas $out/bin/ptxas
+    mkdir -p $out/${python.sitePackages}/jaxlib/cuda/bin
+    ln -s ${cudaPackagesGoogle.cuda_nvcc.bin}/bin/ptxas $out/${python.sitePackages}/jaxlib/cuda/bin/ptxas
```

Comment:
Nit: let's not constrain our future self and use `getBin`

Reply:
done in https://github.com/NixOS/nixpkgs/pull/288829/commits/cdb1b0179d40b3d6c0b398946a5af112ef7ac6f5